cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml

```diff
@@ -690,6 +685,8 @@ Resources:
               - 'sqs:DeleteQueue'
               - 'sqs:CreateQueue'
               - 'sns:Subscribe'
+              - 'sns:ListSubscriptions'
```

Comment:
I'm not sure I understand the feature much, but is this going to be conditional for multiaccount tags? Is there a case where customers who dont need can opt out?